ui/src/app/pages/login-page/login-page.component.ts

```diff
@@ -25,8 +30,13 @@ export class LoginPageComponent implements OnInit {
 	ngOnInit() {
 	}
 
-
 	public async tryLogin(login: string, password: string): Promise<void> {
-		this._userService.login(login, password);
+		await this._userService.login(login, password, () => {
+			loadUserInfo(this._userService, this._store, this._redirectToMain.bind(this));
```

Comment:
Some solutions how you can handle success actions https://stackoverflow.com/questions/43226681/how-to-subscribe-to-action-success-callback-using-ngrx-and-effects
https://netbasal.com/listening-for-actions-in-ngrx-store-a699206d2210

Reply:
need use dispatcher?!

Reply:
Yep. You need to use it to send data to store.

Reply:
You do not need dispatcher for that. You need a subscription.